src/TabbedRoutes.tsx

```diff
@@ -87,6 +89,10 @@ export default function TabbedRoutes() {
   const isProfileButtonDisabled = location.pathname.startsWith("/profile");
   const isSearchButtonDisabled = location.pathname.startsWith("/search");
 
+  useEffect(() => {
```

Comment:
@burakcan You know more than I do about redux. Is there a better way to bootstrap user-specific settings like favorites? Faves are in the communitySlice but we can move them if they should be somewhere else (I like what you did for appearance slice bootstrapping)

Reply:
@aeharding we can listen to actions from other slices in the communitySlice. So when the auth happens (I think the action was called something userChange?) we can listen for it in the communitySlice via an `additionalReducer`. On mobile now so couldn't be so detailed

Reply:
@burakcan Hmmm, I still don't have my head completely wrapped around `extraReducers`

I'm going to merge the PR as is right now, but if you have time, I'd appreciate a PR fixing this!

Reply:
@aeharding I'll try take a look one of these days. It's my last week before vacation on my day-job so it's a bit hectic for me timewise :)